src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -1718,9 +1718,8 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         return _settings->HasUnfocusedAppearance();
     }
 
-    void ControlCore::AdjustOpacity(const int32_t& opacity, const bool& relative)
+    void ControlCore::AdjustOpacity(const double opacityAdjust, const bool relative)

```

Comment:
I wrote the reason for this in the commit description:
> It includes an additional change in order to make the AdjustOpacity setter consistent and symmetric with the Opacity getter.

Let me know what you all think. 🙂